From e762a0675e9e26564eb811bee351a289d6ecba1b Mon Sep 17 00:00:00 2001
From: Hongcheng Xie <hongcheng.xie@intel.com>
Date: Tue, 17 Nov 2020 14:22:29 +0800
Subject: [PATCH] Don't sleep 1s even we don't enforce vintf

Don't always warning and sleep 1s even we don't support
treble and enforce vintf, otherwise it will cause long
booting time and runtime performance issue.

Tracked-On: ACP-1374
Change-Id: I68c42a72624fc6f9df69707ea3fc2363c1ec517c
Signed-off-by: Hongcheng Xie <hongcheng.xie@intel.com>
---
 transport/ServiceManagement.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/transport/ServiceManagement.cpp b/transport/ServiceManagement.cpp
index a7e9626..6f29a6d 100644
--- a/transport/ServiceManagement.cpp
+++ b/transport/ServiceManagement.cpp
@@ -758,7 +758,7 @@ sp<::android::hidl::base::V1_0::IBase> getRawServiceInternal(const std::string&
     const bool allowLegacy = !kEnforceVintfManifest || (trebleTestingOverride && kDebuggable);
     const bool vintfLegacy = (transport == Transport::EMPTY) && allowLegacy;
 
-    if (!kEnforceVintfManifest) {
+    if (!kEnforceVintfManifest && false /* we don't support treble and enforce vintf */) {
         ALOGE("getService: Potential race detected. The VINTF manifest is not being enforced. If "
               "a HAL server has a delay in starting and it is not in the manifest, it will not be "
               "retrieved. Please make sure all HALs on this device are in the VINTF manifest and "
-- 
2.17.1

